src/domain/Avatars/AvatarEntity/AvatarEntity.tsx

```diff
@@ -126,15 +128,11 @@ class AvatarEntity extends React.PureComponent<IAvatarEntityProps> {
     }
 
     return (
-      <Text
-        isInline={false}
-        isTruncated
-        color={Variables.Color.n700}
-        margins={{ left: size === Size.Normal ? 48 : 38 }}
-        type={size === Size.Normal ? Props.TypographyType.Small : Props.TypographyType.XSmall}
+      <StyledTertiaryText
+        size={size}
       >
         {tertiaryText}
-      </Text>
+      </StyledTertiaryText>
     )
```

Comment:
This isn't what I meant in my comment... 
I expected something like this:
```
<TertiaryTextWrapper size>
  <Text
    isTruncated
    color
    type
  >
    {tertiaryText}
  </Text>
</TertiaryTextWrapper>
```

where TertiaryTextWrapper is:
```
const TertiaryTextWrapper = styled.div`
  margin-left: size === Size.Normal ? 48 : 38
`
```

Reply:
Okay I got it now...